curvefs/conf/client.conf

```diff
@@ -92,7 +93,7 @@ s3.baseSleepUs=500
 
 # TODO(huyao): use more meaningfull name
 # background thread schedule time
-s3.intervalSec=3
+s3.intervalSec=0
```

Comment:
Why change this value to 0

Reply:
this feature maybe disable later by huyao